Sources/Private/Model/ShapeItems/ShapeTransform.swift

```diff
@@ -115,6 +115,15 @@ final class ShapeTransform: ShapeItem {
   /// Skew Axis
   let skewAxis: KeyframeGroup<LottieVector1D>
 
+  /// Whether or not this transform has a non-zero skew value
+  var hasSkew: Bool {
+    guard !skew.keyframes.isEmpty, !skewAxis.keyframes.isEmpty else {
+      return false
+    }
+
+    return skew.keyframes.contains(where: { $0.value.cgFloatValue != 0 })
```

Comment:
Should `skewAxis.keyframes` also be checked with contains here?

Reply:
No -- the sample animation added in this PR has a skew of 30 and a skew axis of 0. So having a non-zero axis is not a requirement for rendering a skew.